packages/chrome/src/elements/sheet/Sheet.tsx

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, {
+  useRef,
+  useMemo,
+  useState,
+  HTMLAttributes,
+  PropsWithoutRef,
+  RefAttributes,
+  ForwardRefExoticComponent
+} from 'react';
+import PropTypes from 'prop-types';
+import { useUIDSeed } from 'react-uid';
+import mergeRefs from 'react-merge-refs';
+import { CSSTransition } from 'react-transition-group';
+
+import { StyledSheet, StyledSheetWrapper } from '../../styled';
+import { SheetContext } from '../../utils/useSheetContext';
+import { useFocusableMount } from '../../utils/useFocusableMount';
+
+import { SheetTitle } from './components/Title';
+import { SheetDescription } from './components/Description';
+import { SheetHeader } from './components/Header';
+import { SheetBody } from './components/Body';
+import { SheetFooter } from './components/Footer';
+import { SheetFooterItem } from './components/FooterItem';
+import { SheetCloseButton } from './components/Close';
+
+interface IStaticSheetExport<T, P>
+  extends ForwardRefExoticComponent<PropsWithoutRef<P> & RefAttributes<T>> {
+  Title: typeof SheetTitle;
+  Description: typeof SheetDescription;
+  Header: typeof SheetHeader;
+  Body: typeof SheetBody;
+  Footer: typeof SheetFooter;
+  FooterItem: typeof SheetFooterItem;
+  Close: typeof SheetCloseButton;
+}
+
+export interface ISheetProps extends HTMLAttributes<HTMLElement> {
+  /** An ID that is applied to Sheet elements */
+  id?: string;
+  /** Determines whether the Sheet is open or not **/
+  isOpen?: boolean;
+  /** Determines whether animation for opening and closing the Sheet is used **/
+  isAnimated?: boolean;
+  /** Focuses on the Sheet when isOpen is true and mounted **/
+  focusOnMount?: boolean;
+  /** Restores focus to the previous element after the Sheet is closed **/
+  restoreFocus?: boolean;
+  /** Determines the position of the Sheet **/
+  placement?: 'start' | 'end';
+  /** Sets the width in pixels, based on the placement of the Sheet */
+  size?: number;
+}
+
+/**
+ * @extends HTMLAttributes<HTMLElement>
+ */
+// eslint-disable-next-line react/display-name
+export const Sheet = React.forwardRef<HTMLElement, ISheetProps>(
+  ({ id, isOpen, isAnimated, focusOnMount, restoreFocus, size, children, ...props }, ref) => {
+    const sheetRef = useRef<HTMLElement>(null);
+
+    const seed = useUIDSeed();
+    const [idPrefix] = useState<string>(id || seed(`sheet_${PACKAGE_VERSION}`));
+    const titleId = `${idPrefix}--title`;
+    const descriptionId = `${idPrefix}--description`;
+
+    const sheetContext = useMemo(() => ({ idPrefix }), [idPrefix]);
+
+    useFocusableMount({ targetRef: sheetRef, isMounted: isOpen, focusOnMount, restoreFocus });
```

Comment:
What was the conclusion regarding sighted keyboard only users and scrolling body text?

Reply:
The consideration is out-of-scope for the initial round. If this behavior is critical (it very well may be), then Garden will take a step back and re-evaluate a11y for potential scroll containers (`DrawerModal`, `TabPanel`, `Accordion.Panel`, modal `Body`, `Stepper.Content`, etc.). Otherwise, the expectation is for the consumer to implement the [useScrollRegion](https://github.com/zendeskgarden/react-containers/tree/main/packages/scrollregion) hook.